README.md

```diff
@@ -229,6 +230,7 @@ render() {
 | `resizeDebounce` | `number` | :no_entry_sign: | `250` | Debounce time in milliseconds |
 | `rootTag` | `string` | :no_entry_sign: | `'ul'` | Tag to be rendered as the root element of the component |
 | `totalSlides` | `number` | :heavy_check_mark: | :no_entry_sign: | Total value of sliders that will be rendered |
+| `minPerPage` | `number` | :no_entry_sign: | `1` | This prop works the same way the `minPerPage` of `Slider` and this component should receive the same value of `Slider` |
```

Comment:
```suggestion
| `minItemsPerPage` | `number` | :no_entry_sign: | `1` | This prop works the same way the `minPerPage` of `Slider` and this component should receive the same value of `Slider` |
```

Reply:
@iaronaraujo since on the `Slider` component the prop is already called just `perPage`, I think it's fine for it to be called just `minPerPage`

Reply:
Yep, I did not want to call it `minItemsPerPage` here as well because I wanted to maintain the similarity with `perPage` prop. 

The user of the Shelf component does not need to know about how this prop is received in the `Slider` component, I think they are completely independent. That said, I don't think it's a problem to have it called `minPerPage` here and `minItemsPerPage` in the `Shelf` if that's what you thought about :)